File: app/src/main/java/com/shopcart/Product.java
package com.shopcart;

import java.util.ArrayList;

public class Product {

    private String id;
    private String name;
    private Double price;
    private ArrayList<String> photosUrl;

    public Product() {
    }

    public Product(String name, Double price, ArrayList<String> photosUrl) {
        this.name = name;
        this.price = price;
        this.photosUrl = photosUrl;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }

    public ArrayList<String> getPhotosUrl() {
        return photosUrl;
    }

    public void setPhotosUrl(ArrayList<String> photosUrl) {
        this.photosUrl = photosUrl;
    }
}

File: app/src/main/java/com/shopcart/data/models/Product.kt
package com.shopcart.data.models

data class Product(
    val id: String = "",
    val category: String = "",
    val name: String = "",
    val description: String = "",
    var quantity: Int = 0,
    val price: Double = 0.0,
    val offer: String = "",
    val photosUrl: ArrayList<String> = ArrayList(),
    val isFeatured: Boolean = false,
    val isBestSell: Boolean = false,
    val isFavorite: Boolean = false
)



